cli/internal/cmd/init.go

```diff
@@ -150,23 +168,26 @@ func initialize(cmd *cobra.Command, newDialer func(validator *cloudcmd.Validator
 		}
 		return err
 	}
-
+	i.log.Debugf("Writing Constellation id file")
 	idFile.CloudProvider = provider
-	if err := writeOutput(idFile, resp, cmd.OutOrStdout(), fileHandler); err != nil {
+	if err := i.writeOutput(idFile, resp, cmd.OutOrStdout(), fileHandler); err != nil {
 		return err
 	}
 
 	return nil
 }
 
-func initCall(ctx context.Context, dialer grpcDialer, ip string, req *initproto.InitRequest) (*initproto.InitResponse, error) {
+func (i *initCmd) initCall(ctx context.Context, dialer grpcDialer, ip string, req *initproto.InitRequest) (*initproto.InitResponse, error) {
 	doer := &initDoer{
 		dialer:   dialer,
 		endpoint: net.JoinHostPort(ip, strconv.Itoa(constants.BootstrapperPort)),
 		req:      req,
+		log:      i.log,
 	}
+	i.log.Debugf("Making initialization call, doer is %+v", doer)
 	retrier := retry.NewIntervalRetrier(doer, 30*time.Second, grpcRetry.ServiceIsUnavailable)
 	if err := retrier.Do(ctx); err != nil {
+		i.log.Debugf("Initialization call errored, retrying...")
 		return nil, err
 	}
```

Comment:
```suggestion
	retrier := retry.NewIntervalRetrier(doer, 30*time.Second, grpcRetry.ServiceIsUnavailable)
	if err := retrier.Do(ctx); err != nil {
		return nil, err
	}
```
Retries only happen inside the `retrier.Do()` function, errors returned here usually mean hard failures.